Lib/test/test_move.py

```diff
@@ -0,0 +1,282 @@
+import unittest
+import sys
+import inspect
+
+global_var_1 = 0
+global_var_2 = 2
+
+# This class tests the move of local variables, these are called `NAME`
+# in Pythons grammar.
+class TestNameMoves(unittest.TestCase):
+
+    def test_move_local_basic(self):
+        x = 42
+        y = move x
+        self.assertEqual(y, 42)
+        # Check that x has been deleted from locals
+        self.assertNotIn("x", locals())
+
+    def test_move_local_as_arg(self):
+        def take_one(x):
+            self.assertEqual(x, "Super secret value")
+
+        y = "Super secret value"
+        take_one(move y)
+
+        # Check that y has been deleted from locals
+        self.assertNotIn("y", locals())
+
+    def test_read_local_before_move(self):
+        def take_several(cp, mv):
+            self.assertEqual(cp, "Mister Flesh Stick")
+            self.assertEqual(mv, "Mister Flesh Stick")
+
+        x = "Mister Flesh Stick"
```

Comment:
"Mister Flesh Stick"?

Reply:
It's a borderlands 2 reference, like most of the names in this file. It's a good game :D